### DISCUSSION TITLE
Teslamate running on Raspberry Pi 3+ under bookworm OS version?

### DISCUSSION THREAD

**Discussion by DrMichael** [teslamate-org/teslamate#3584] [General]
Hi

I am wondering, whether I can upgrade my RPi3+ to the latest OS version.
Any experiences with this?

Michael

**Comment by cwanja** [comment]
I am running Bullseye without any issues. I saw there was a release earlier this month on their website - have not tested. But if it's incremental, I imagine it's fine. 

https://www.raspberrypi.com/software/operating-systems/

**Reply by dsnowdy** [reply]
My install totally crapped out months ago so I wanted to rebuild it. Running docker v2 and Bookworm on 3+. Seems to be working fine.
